build-logic/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt

```diff
@@ -185,6 +186,7 @@ abstract class ExternalModulesExtension {
         bouncycastlePgp to License.MIT,
         bouncycastleProvider to License.MIT,
         bsh to License.Apache2,
+        capsule to License.BSDStyle,
```

Comment:
Is that enough to have it visible in the docs and / or distribution as required by the license?

Reply:
I don't know. @blindpirate, do you know?